app/views/prior_authority/steps/primary_quote/edit.html.erb

```diff
@@ -9,12 +9,32 @@
 
     <%= step_form @form_object do |form| %>
       <%= suggestion_select form, :service_type, PriorAuthority::QuoteServices.values, :to_sym, :translated, width: 20,
-                            options: { include_blank: true }, label: { size: 'm' }, hint: { size: 's', text: t('.service_name_hint')},
+                            options: { include_blank: true }, label: { size: "m" }, hint: { size: "s", text: t(".service_name_hint")},
                             data: { autoselect: false } %>
       <h2 class="govuk-heading-m"><%= t(".service_details") %></h2>
-      <%= form.govuk_text_field :contact_full_name, label: { text: t(".contact_full_name"), size: 's' } %>
-      <%= form.govuk_text_field :organisation, label: { text: t(".organisation"), size: 's' } %>
-      <%= form.govuk_text_field :postcode, label: { text: t(".postcode"), size: 's' } %>
+      <%= form.govuk_text_field :contact_full_name, label: { text: t(".contact_full_name"), size: "s" } %>
+      <%= form.govuk_text_field :organisation, label: { text: t(".organisation"), size: "s" } %>
+      <%= form.govuk_text_field :postcode, label: { text: t(".postcode"), size: "s" } %>
+      <%= form.govuk_file_field :document, label: { text: t(".file_upload"), size: "m" }, hint: { text: t(".file_upload_hint") }, accept: SupportedFileTypes::QUOTE_DOCUMENT.join(',') %>
```

Comment:
You'll also need to add this file to app/views/prior_authority/steps/primary_quote_summary/_service_cost_card.html.erb